### PR TITLE
[PR] Channel Manager API

### PR DESCRIPTION
**Pull request by ondrahermanek**
Can be viewed here - https://github.com/MewsSystems/public/blob/chm-ipi/content/developers/api/channelManager.md

It would be really great, if someone could fix typos - @MeissnerJan ? You need to understand the API anyway.

### CODE REVIEW

File: content/developers/api/channelManager.md [hunk 1]
```diff
@@ -0,0 +1,1099 @@
+---
+title: Channel Manager API (v1)
+---
+
+The connection between Mews (as PMS) and a Channel Manager is a 2-way connecton. This document defines both Mews (as PMS) side and defines an API on Channel Manager side as well. The Mews side mainly accepts bookings from Channel Manager and the Channel Manager side accepts inventory updates from PMS and distributes them to connected OTAs.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: Pls 1 sentence per line. It makes the diff more readable when you do changes.

Comment (jirihelmich) [reply content/developers/api/channelManager.md]: The general description is missing some basic description of the 2-way mechanism. Websockets? Webhooks? A pigeon? :-)

Comment (jirihelmich) [reply content/developers/api/channelManager.md]: Pls reformat the PR to a sentence per line, it's hard to comment on paragraphs.

**Comment by siroky** [comment]
Merged because I'm restructuring documentation, but it should be reviewed anyway.

File: content/developers/api/channelManager.md [hunk 2]
```diff
@@ -0,0 +1,1127 @@
...
+    	- [Update prices](#update-prices)
+    	- [Update availability](#update-availability)
+    	- [Update restrictions](#update-restrictions)
+    	- [Confirm booking](#confirm-booking)
+    	- [Change notification](#change-notification)
+
+
+### General Remarks
+
+All operations of the API require `AccessToken` to be present in the request, all operations regarding an established connection require `ConnectionCode` as well. `AccessToken` determines a concrete Channel Manager, `ConnectionCode` determines a concete connection between the Channel Manager and a hotel.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: typo, concete

File: content/developers/api/channelManager.md [hunk 3]
```diff
@@ -0,0 +1,1127 @@
...
+    	- [Update restrictions](#update-restrictions)
+    	- [Confirm booking](#confirm-booking)
+    	- [Change notification](#change-notification)
+
+
+### General Remarks
+
+All operations of the API require `AccessToken` to be present in the request, all operations regarding an established connection require `ConnectionCode` as well. `AccessToken` determines a concrete Channel Manager, `ConnectionCode` determines a concete connection between the Channel Manager and a hotel. 
+There can be multiple connections between one hotel and one Channel Manager. 
+In each environment, the `AccessToken` will be provided to you by Mews. For development purposes, use the  [Develop Environment](#develop-environment).
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: 2 sentences on a line

File: content/developers/api/channelManager.md [hunk 4]
```diff
@@ -0,0 +1,1127 @@
...
+
+The documentation uses term **Propetry** to describe a client of Channel Manager or PMS and **Space Type** to reffer to a space sold/offered by the **Property**. 
+In hospitality world, a **Property** is a hotel (hostel) and **Space Type** is a room (bed). But **Property** can be some sport facilty and then the **Space Type** would refer to a squash court, that is why those unusual, yet general, terms. 
+
+### Responses
+
+Each API call is expected to return a response. 
+The **Http status code** of response will be `200` in case the API call processing was successfull as well as if the API call processing failed.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: Failed - I would explain what kind of fail that is.

File: content/developers/api/channelManager.md [hunk 5]
```diff
@@ -0,0 +1,1127 @@
...
+### Responses
+
+Each API call is expected to return a response. 
+The **Http status code** of response will be `200` in case the API call processing was successfull as well as if the API call processing failed. 
+In later case, the error will be provided in the response. 
+
+#### Plain reponse
+
+This response object representing the basic response. 
+In case there is more complex more complex response expected, that response object will extend this simple response object.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: double more complex

File: content/developers/api/channelManager.md [hunk 6]
```diff
@@ -0,0 +1,1127 @@
...
+
+This part defines the API on Mews side. 
+
+### Mews API Modes
+There are 3 modes in which a Channel Manager can use the Mews API
+
+#### Fully Automated
+
+Mews allows any Channel manager to fully automate the process of setting up a connecting for any property. 
+This process doesn't require any Channel Manager Support person to overview the connection set up, it can be done just by the employee
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: dot missing

File: content/developers/api/channelManager.md [hunk 7]
```diff
@@ -0,0 +1,1127 @@
...
+
+Once property employee, who will be managing the property connection, creates an account in the Channel manager, they can start connecing Mews properties they work in.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: connecTing

File: content/developers/api/channelManager.md [hunk 8]
```diff
@@ -0,0 +1,1127 @@
...
+### Mews API Modes
+There are 3 modes in which a Channel Manager can use the Mews API
+
+#### Fully Automated
+
+Mews allows any Channel manager to fully automate the process of setting up a connecting for any property. 
+This process doesn't require any Channel Manager Support person to overview the connection set up, it can be done just by the employee
+
+Once property employee, who will be managing the property connection, creates an account in the Channel manager, they can start connecing Mews properties they work in. 
+It is recommended, that the emloyee uses the email he uses to log in to Mews and that Channel manager verifies the email address.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: emPloyee

File: content/developers/api/channelManager.md [hunk 9]
```diff
@@ -0,0 +1,1127 @@
...
+
+1. Channel manager calls [Get Properties](#get-properties) to obtain list of properties that can be connected.
+2. Employee select whether he wants to create a new connection or use already prepared connection in Mews.
+	- Channel manager calls [Create Connection](#create-connection) to create a new connection with selected property. 
+	- Channel manager calls [Get Configuration](#get-configuration) to use exsting connection. 
+3. In case Channel manager allows to employee change the property inventory (i.e. rate plan - space type relations), Channel manager call `[Set Inventory](#set-inventory)` to propagate the changes to Mews
+4. Channel Manager calls [Request ARI Update](#request-ari-update) so employee can verify that the data sent from Mews are correctly accepted by the Channel Manager.
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: emPloyee

File: content/developers/api/channelManager.md [hunk 10]
```diff
@@ -0,0 +1,1127 @@
...
+5. Once data is confirmed by the employee, the Channel Manager continues with testing the reservation delivery by [Process Group](#process-group) calls.
+6. Once reservation delivery is confirmed by employee, the Channel Manager confirms the connection is set up by calling [Update Connection](#create-connection).
+
+#### Baisc
```
Comment (jirihelmich) [inline content/developers/api/channelManager.md]: Basic not Baisc

**Pull request merged**